seed/challenges/02-javascript-algorithms-and-data-structures/basic-data-structures.json

```diff
@@ -813,11 +809,11 @@
         "console.log(addFriend(user, 'Pete'));"
       ],
       "tests": [
-        "assert('name' in user && 'age' in user && 'data' in user, 'message: The <code>user</code> object has <code>name</code>, <code>age</code>, and <code>data</code> keys');",
-        "assert((function() { let L1 = user.data.friends.length; addFriend(user, 'Sean'); let L2 = user.data.friends.length; return (L2 === L1 + 1); })(), 'message: The <code>addFriend</code> function accepts a <code>user</code> object and a <code>friend</code> string as arguments and adds the friend to the array of <code>friends</code> in the <code>user</code> object');",
-        "assert.deepEqual((function() { delete user.data.friends; user.data.friends = ['Sam', 'Kira', 'Tomo']; return addFriend(user, 'Pete') })(), ['Sam', 'Kira', 'Tomo', 'Pete'], 'message: <code>addFriend(user, \"Pete\")</code> should return <code>[\"Sam\", \"Kira\", \"Tomo\", \"Pete\"]</code>');"
+        "assert('name' in user && 'age' in user && 'data' in user, 'The user object has name, age, and data keys');",
+        "assert((function() { let L1 = user.data.friends.length; addFriend(user, 'Sean'); let L2 = user.data.friends.length; return (L2 === L1 + 1); })(), 'The addFriend function accepts a user object and a friend as arguments and adds the friend to the array of friends in the user object.');"
       ],
```

Comment:
The below tests need to be switched around (sorry, the first review may have been a little unclear), the 3 showing as removed are the latest work, they are properly formatted with code tags and the instructions will not match the tests if the third test is omitted. 

Just for the record, even though EverNote may not be doing so well, I love this tool that you recommended!

![image](https://cloud.githubusercontent.com/assets/18563015/22490876/c40cf2a6-e7ed-11e6-9c04-bd07fcf0e936.png)

Other than that, everything looks good to me, and this should be a simple change. I would fix it myself, but not sure how to add to your PR without write access to your repo.